File: Utility/WinProcess.hs
{- Windows processes
 -
 - Copyright 2014 Joey Hess <id@joeyh.name>
 -
 - License: BSD-2-clause
 -}

module Utility.WinProcess where

import Utility.PID

terminatePID :: PID -> IO ()
terminatePID p = warning "terminating processes on windows is not currently working"

File: Utility/winprocess.c
#include <windows.h>

void terminatepid (DWORD pid) {
	HANDLE h;
	h = OpenProcess(PROCESS_TERMINATE, 0, pid);
	if (h != NULL) {
		TerminateProcess(h, 1);
	}
	CloseHandle(h);
}
